mappings/maestro_file_centric.json

```diff
@@ -0,0 +1,239 @@
+  },
+  "settings": {
+    "index.number_of_shards": 3,
+    "index.max_results_window": 300000,
+    "analysis": {
+      "analyzer": {
+        "autocomplete_analyzed": {
+          "filter": ["lowercase", "edge_ngram"],
+          "tokenizer": "standard"
+        },
+        "autocomplete_prefix": {
+          "filter": ["lowercase", "edge_ngram"],
+          "tokenizer": "keyword"
+        },
+        "lowercase_keyword": {
+          "filter": ["lowercase"],
+          "tokenizer": "keyword"
+        }
+      },
+      "filter": {
+        "edge_ngram": {
+          "max_gram": 20,
+          "min_gram": 20,
+          "side": "front",
+          "type": "edge_ngram"
+        }
+      }
+    }
+  },
+  "mappings": {
+    "dynamic": false,
+    "date_detection": false,
+    "properties": {
+      "file_autocomplete": {
+        "fields": {
+          "analyzed": {
+            "analyzer": "autocomplete_prefix",
+            "search_analyzer": "lowercase_keyword",
+            "type": "text"
+          },
+          "lowercase": {
+            "analyzer": "autocomplete_prefix",
+            "type": "text"
+          },
+          "prefix": {
+            "analyzer": "autocomplete_prefix",
+            "search_analyzer": "lowercase_keyword",
+            "type": "text"
+          }
+        },
+        "type": "keyword"
+      },
+      "study_id": {
+        "type": "keyword",
+        "copy_to": ["file_autocomplete"]
+      },
+      "object_id": {
+        "type": "keyword",
+        "copy_to": ["file_autocomplete"]
+      },
+      "file_type": {
+        "type": "keyword"
+      },
+      "data_type": {
+        "type": "keyword",
+        "copy_to": ["file_autocomplete"]
+      },
+      "file_access": {
+        "type": "keyword"
+      },
+      "analysis": {
+        "properties": {
+          "analysis_id": {
+            "type": "keyword",
+            "copy_to": ["file_autocomplete"]
+          },
+          "analysis_type": {
+            "type": "keyword"
+          },
+          "analysis_version": {
+            "type": "integer"
+          },
+          "experiment": {
+            "type": "object",
+            "properties": {
+              "platform": {
+                "type": "keyword"
+              },
+              "platform_model": {
+                "type": "keyword"
+              },
+              "sequencing_date": {
+                "type": "keyword"
+              },
+              "library_name": {
+                "type": "keyword"
+              },
+              "library_strategy": {
+                "type": "keyword"
+              },
+              "sequencing_center": {
+                "type": "keyword"
+              }
+            }
+          }
+        }
+      },
+      "dynamic": {
+        "properties": {
+          "workflow_name": {
+            "type": "keyword"
+          },
+          "workflow_type": {
+            "type": "keyword"
+          },
+          "analysis_software": {
```

Comment:
where does this  come from?

Reply:
some of these are based on draft mapping you had a year or less ago, so some fields are outdated

Reply:
@junjun-zhang after reviewin ICGC and GDC, this is something that is currently missing from SONG payload.  while we have the workflow name, we dont include teh software used (like BWA MEM for alignment).

Reply:
right, since SONG analysis type schemas are more fleshed out, should be good to take what's currently in SONG

Reply:
yes, @rosibaj we discussed about this in bioinfo meetings, decided not to include tool level information in the SONG workflow section

Reply:
@christinayung do we want the tool level information on the platform as a searchable feature?

Reply:
Since there can be many tools with changing version, the list may be hard to maintain.  I'd suggest storing the workflow version so that the tools can be inspected at github in details.